src/components/TopNav.js

```diff
@@ -1,21 +1,62 @@
+import { useState, useEffect } from 'react';
 import AppBar from '@material-ui/core/AppBar';
+import Button from '@material-ui/core/Button';
+import i18n from 'i18next';
+
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   navbar: {
-    height: 50,
+    height: 60,
     display: 'flex',
-    justifyContent: 'center',
+    flexDirection: 'row',
+    alignContent: 'center',
+    justifyContent: 'space-between',
     alignItems: 'center',
   },
+  title: {
+    marginLeft: 30,
+  },
+  button: {
+    marginRight: 10,
+  },
 }));
 
 export default function TopNav() {
   const classes = useStyles();
 
+  const [language, setLanguage] = useState(localStorage.getItem('lng') || 'en');
```

Comment:
please use `en` from `useTranslation()` variables it should have the default fallback language set